tests/unit/type/string.tcl

```diff
@@ -102,6 +102,75 @@ start_server {tags {"string"}} {
         assert_equal 20 [r get x]
     }
 
+    test "GETEX EX option" {
+        r del foo
+        r set foo bar
+        r getex foo ex 10
+        set ttl [r ttl foo]
+        assert {$ttl <= 10 && $ttl > 5}
```

Comment:
FYI, there's an advantage of using `assert_equal` and `assert_range`, in that when the assertion fails, you always get the offensive value (i.e. seeing `Expected '$value' to be between to '$min' and '$max'`)
if you do that, you don't need to store the value into an argument.
so it'll look like
`assert_range [r ttl foo] 5 10`

Reply:
Thank you! this was helpful updated it in all of the test.